.github/actions/sign-attestations/dist/rekor.pub

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
so this is not automatically generated, nice!

Reply:
it is!